core/version/gear.ex

```diff
@@ -10,7 +10,10 @@ defmodule AntikytheraCore.Version.Gear do
   require AntikytheraCore.Logger, as: L
 
   @installed_gear_ratio_threshold 0.5
-  @notify_threshold Application.fetch_env!(:antikythera, :gear_install_notify_threshold_in_seconds)
+  @notify_threshold Application.fetch_env!(
+                      :antikythera,
+                      :gear_install_notify_threshold_in_seconds
+                    )
```

Comment:
In the last pull request I forgot `mix format`. I will split it into other PR if necessary.